rj_gameplay/rj_gameplay/tactic/nmark_tactic.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, n: int):
         self.num_markers = n
 
         # create empty mark SkillEntry for each robot
-        self.mark_list = [
+        self.mark_list: type[mark.Mark] = [
```

Comment:
Error: rj_gameplay/rj_gameplay/tactic/nmark_tactic.py:82: error: Need type annotation for 'mark_list'
-> Added type hinting from rj_gameplay.skill.mark.Mark